src/plots/barPlot.ts

```diff
@@ -518,6 +545,10 @@ export module Plots {
       return barPixelWidth;
     }
 
+    private _updateBarPixelWidth() {
+      this._barPixelWidth = this._getBarPixelWidth();
```

Comment:
if I add that line in the `_getBarPixelWidth()` it does not really make much sense from a reading stand point. Also, can't rename or remove old function